internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -210,7 +210,15 @@ func resourceKustoCluster() *pluginsdk.Resource {
 				Computed: true,
 			},
 
-			"zones": azure.SchemaZones(),
+			"zones": {
```

Comment:
is there a reason we are using a set here and not the pre existing zones functions? unlike other resources does order not matter here?

Reply:
I think order doesn't matter in any resource's zones. Maybe I should change it in azure.SchemaZones()?

Reply:
The order _should_ matter in some cases, for example DB replicas - however that'd likely change service-to-service and that information isn't available in the Swagger

Reply:
Hi @katbyte , is this PR ready to go?

Reply:
wrt to toms concern about the order of the zones, can you confirm order doesn't matter?

Reply:
Sure, I'll contact service team to make sure of it.

Reply:
@katbyte , just confirmed with service team, the order doesn't matter here.